weasyl/templates/help/ratings.html

```diff
@@ -181,15 +180,11 @@ <h5>Audiences Over the Age of Eighteen</h5>
             <th>Drug Use</th>
             <td>Equivalent to Mature</td>
           </tr>
+          <tr>
+            <th>Kinks/Fetishes</th>
+            <td>Any “hard” kink or fetish is rated Explicit, including but not limited to: scat/watersports (urine or feces), including blowout/leaky diapers, vomit, breast milk, bondage and BDSM, hypnosis, extreme blood, gore, sexually-explicit transformation, diaper fetishism (with a sexual component), and other things not listed here. A comprehensive list is not possible, but the rule of thumb is if there is urination/defecation or explicit sexual activity involved, then it is rated Explicit.</td>
```

Comment:
As above, I think we're at risk of being drawn into "what fetishes are hard fetishes" and arguments over "but it's just clothing" etc. I'm not sure what the solution is but I'd rather we not dictate to someone that their interest in whatever-thing must be sexual because it /might/ be sexual to someone.